TransparentWall/HarmonyPatches/Patches/ObstacleControllerCullingLayer.cs

```diff
@@ -27,13 +27,12 @@ private static void Postfix(ref ObstacleController __instance)
                 Camera.main.cullingMask &= ~(1 << Configuration.WallLayerMask);
             }
 
-            if (Configuration.EnableForHeadset || Configuration.DisableForLivCamera)
+            if (!Configuration.EnableForHeadset && !Configuration.DisableForLivCamera) return;
+
+            var mesh = __instance.gameObject?.GetComponentInChildren<Renderer>(false);
```

Comment:
I'd appreciate if you didn't use "var" as is does not conform with the coding style of this project
I find them lazy 😋

```suggestion
            Renderer mesh = __instance.gameObject?.GetComponentInChildren<Renderer>(false);
```